src/tools/config.rs

```diff
@@ -0,0 +1,46 @@
+use std::io;
+
+use crate::{protocol::codecs::binary::Payload, setup::config::NodeConfig};
+
+/// Test configuration. Contains setup options for [SyntheticNode], [Node] and [pea2pea::Config].
```

Comment:
have you checked that `SyntheticNode` and `Node` are linked correctly when using `cargo doc`? I don't see them in the scope